### DISCUSSION TITLE
How to create a full AzuraCast backup that can be restored to a different machine?

### DISCUSSION THREAD

**Discussion by renatopinto89** [AzuraCast/AzuraCast#5125] [General]
Hi guys, first off all, thank you so much for this amazing project.

So here's the thing, I have one server running AzuraCast 0.14.1 which doesn't have performance issues and I want to find a way to fully back it up in order to install it on a new server, since using the normal installation methods will always install the latest 0.15 stable release and I'm trying to avoid the rolling ones.

Is there any way of doing that without being snapshot the entire server?

Thank you.

**Comment by ItsMitchh** [comment]
Generally we dont give advise on installing older versions because they dont contain all the up to date security and bug fixes as the latest releases do. Backups and snapshots are your best bet.
